src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-pyttsx3.py

```diff
@@ -0,0 +1,19 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2020 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the GNU General Public
+# License (version 2.0 or later).
+#
+# The full license is available in LICENSE.GPL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+# ------------------------------------------------------------------
+
+# pyttsx3 conditionally imports drivers module based on specific platform.
+# https://github.com/nateshmbhat/pyttsx3/blob/5a19376a94fdef6bfaef8795539e755b1f363fbf/pyttsx3/driver.py#L40-L50
+
+
+from PyInstaller.utils.hooks import collect_submodules
+
+hiddenimports = collect_submodules("pyttsx3.drivers")
```

Comment:
Does it make sense to collect all drivers? `sapi5` seems to be Windows-specific, and `nsss` seems macOS-specific. `espeak` should at least nominally be cross-platform, but I guess not so much if it's trying to load `libespeak.so.1` as the shared library... 

So its seems each platform has one actual driver available (in addition to `dummy`).

Reply:
Hmm, well the choice of driver is configurable but it doesn't look like a feature that gets used much - or even one which can be used.